src/components/prod/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
style name should be started from small letter

Reply:
corrected remark

Reply:
where?